tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -313,6 +313,7 @@ tensorflow/lite/micro/kernels/arg_min_max.cc \
 tensorflow/lite/micro/kernels/batch_to_space_nd.cc \
 tensorflow/lite/micro/kernels/cast.cc \
 tensorflow/lite/micro/kernels/ceil.cc \
+tensorflow/lite/micro/kernels/circular_buffer_flexbuffers_generated_data.cc \
```

Comment:
It feels like this shouldn't be in the KERNEL_SRCS, but that's where the other flexbuffers generated data lives, and it does not build if it's only part of the tests.

Reply:
I think adding an explicit test in the Makefile will help here.

Something like:
https://github.com/tensorflow/tensorflow/blob/6a2d982fa971ef803df4adf5dda449431b76ce3e/tensorflow/lite/micro/tools/make/Makefile#L679-L691